deployments/cdi/README.md

```diff
@@ -0,0 +1,16 @@
+# SR-IOV Network Device Plugin in CDI mode deployment
+
+SR-IOV Network Device Plugin supports [Container Device Interface (CDI)](https://github.com/container-orchestrated-devices/container-device-interface).
+
+To enable CDI mode should be started with `--use-cdi` CLI argument. There are also additional volume mounts required
```

Comment:
This is not congruent with [FlagInit()](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/489/files#diff-f1a4e99ad0d1a580dda8274ab56d83450f41088bae0f4d0df0808d03c851d20dR64) where it looks like the feature is enabled by default.
I don't know if it's better to have it enabled by default, but in that case, we have to document the opposite usage (i.e. how to disable it)